index.d.ts

```diff
@@ -35,59 +35,113 @@ declare namespace tempy {
 		*/
 		readonly prefix?: string;
 	};
+
+	/**
+	The path to the temporary object created by the function.
```

Comment:
What is a "temporary object"? Not clear to me.

Reply:
Fixed!